Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Coupled-Transmons/Use Case 4 - Single- and Two-Qubit State and Process Tomography/README.md

```diff
@@ -0,0 +1,185 @@
+
+This use-case has four potential applications, some of which may not be applicable/necessary, namely
+1. Single-qubit state tomography,
+2. Single-qubit process tomography,
+3. Two-qubit state tomography, and
+4. Two-qubit process tomography
+
+The most complex applications are obviously the two-qubit cases, which is what we will focus on below.
+The single-qubit cases follow trivially from them.
+
+## 1. Experimental set-up
+
+<img align="right" src="setup.png" alt="setup" width="400"/>
+
+
+### 1.1 Experimental set-up
+The chip consists of 9 2D flux-tunable transmons arranged into a 3x3 grid, with nearest neighbours connected via flux tunable lines.
+
+For the purposes of this experiment, we focus on two neighbouring flux-tunable transmons (tuned using SQUID non-linear inductors and nearby flux lines) coupled to individual readout resonators with a common feedline, as well as the tunable coupler between them. This system is isolated from the rest of the chip by parking neighbouring qubits far away via external DC sources. See the figure on the right.
+
+The qubits are driven with analog IQ signal output pairs generated by the OPX+ and up-converted with an Octave. Their individual fluxes, as well as the flux of the tunable coupler, are tuned using three additional analog outputs from the OPX+.
+
+The readout resonators are driven with an analog IF signals pair generated by the OPX+ and up-converted with an Octave.
+
+The transmitted signal is measured by the OPX after down-conversion with the Octave, and the response of each resonator extracted by employing a multiplexed readout scheme.
+
+Note: single qubit tomography could be carried out using this chip by focusing on one of the two qubits and parking the second qubit far away.
+
+
+### 1.2 Pre-requisite calibrations
+
+
+For each resonator and qubit:
+
+* The first calibration is to find the readout resonator frequency and readout pulse amplitude using 2D resonator spectroscopy for each resonator.
+* With the frequencies and readout amplitudes set, perform spectroscopy on each resonator while sweeping the flux bias of the corresponding coupled qubit. The resulting 2D "cosine" plots show the resonators' frequencies as functions of the flux biases. From this, the fluxes at the qubits' sweet spots, at which point the qubits are most insensitive to flux noise, can be set. We want to set the flux lines to the qubits' flux insensitive points by default.
+* Next, we calibrate the qubits' $\pi/2$ and $\pi$ pulses. This can generally be done by first performing qubit spectroscopy to get a rough idea of the qubits' frequencies. Next, simple Rabi measurements can be carried out, where both the $pi$ pulse amplitudes and frequencies are scanned while the flux bias is set to the qubit flux insensitive point found previously. The qubits' frequencies can be fine-tuned using Ramsey measurements, and the $\pi$ pulses calibrated further by performing Rabi error amplification experiments.
```

Comment:
maybe let's specify that it's Chevron Rabi by writing "simple Chevron Rabi" ?

Reply:
Sure!